spec/Classes.tex

```diff
@@ -156,8 +156,10 @@ \subsection{Class Fields}
 \index{fields!class}
 
 Variable declarations within a class declaration define
-fields within that class.  Parameter fields make a class generic.
-Variable fields define the storage associated with a class instance.
+fields within that class.
+% When a class is generic is defined in \section{Class Declarations}.
+\chpl{var} fields define the storage associated with a class instance.
```

Comment:
Isn't this also true for const fields? Also, I find the phrase "define the storage associated with a class instance"
awkward. I think I would prefer "are stored in each class instance" or something along those lines.
One alternative would be to sum it up at the end: "The storage for each class instance will
consist of memory for each var or const field". Or something.